x/evm/types/statedb.go

```diff
@@ -88,12 +96,12 @@ type CommitStateDB struct {
 	lock sync.Mutex
 }
 
-// NewCommitStateDB returns a reference to a newly initialized CommitStateDB
+// newCommitStateDB returns a reference to a newly initialized CommitStateDB
```

Comment:
The life cycle of csdb.logSize  is a block

Reply:
yes，I will add it in the next commit